src/components/GameView/GameOverDialog.vue

```diff
@@ -1,26 +1,43 @@
 <template>
   <v-dialog v-model="show" persistent max-width="650">
     <v-card id="game-over-dialog">
-      <v-card-title :data-cy="headingDataAttr">
-        <h1>{{ heading }}</h1>
-      </v-card-title>
-      <v-card-text class="d-flex justify-center">
-        <v-img
-          :src="logoSrc"
-          :data-cy="logoDataAttr"
-          class="logo-image"
-        />
+      <div class="d-flex">
+        <v-card-title :data-cy="headingDataAttr" class="mt-8">
+          <h1 class="dialog-header">{{ heading }}</h1>
```

Comment:
In addition to calling ties 'Draw' instead of stalemate, can we update the heading to specify which game this was or whether you won the match? I'm envisioning the headings reading like:

1. You Win Game 1
2. You Lose Game 2
3. Game 3: Draw
4. You Win the Match

Notice the 4th won doesn't say which game it is, just that you won the match.  I hope this can help players keep track of which game is which and when the match is over (I know we have the paragraph text already but I hope this makes it even more clear).

@lihaobhsfer and @seriouslysean what do you think about this from a UX perspective?

Reply:
How does
```
Game 1: You Win 
Game 2: You Lose
Game 3: Draw
You Win the Match
```
look like? Fewer conditionals to format the message

Reply:
Sounds good